Homework_3.py

```diff
@@ -0,0 +1,46 @@
+# import os
+# import sys
+
+from pyspark import StorageLevel
+from pyspark.sql import SparkSession
+
+# Making spark object
+spark = SparkSession.builder.master("local[*]").getOrCreate()
+
+# creating connection to Mariadb
+database_name = "baseball"
+port = "3306"
+user = "root"
+password = " "
+
+# reading data from jdbc and creating input table
+Input_Table = (
+    spark.read.format("jdbc")
+    .options(
+        url=f"jdbc:mysql://localhost:{port}/{database_name}",
```

Comment:
Please check if this is giving JDBCCompliantTimezoneShift error on your system for UTC. 

Somehow I had to add below arguements to make it working.
"useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC"